backend/local/watcher.coffee

```diff
@@ -29,6 +25,8 @@ class LocalWatcher
     start: (callback) =>
         log.info 'Start watching filesystem for changes'
         @paths = []
+        @pending = Object.create null  # ES6 map would be nice!
```

Comment:
I could, but it's slightly better to use `Object.create null`. For example, if someone add a file called `toString` or `constructor`, it will fail with `@pending = {}`, but not with `Object.create null`.